custom/icds_reports/ucr/data_sources/dashboard/child_tasks.json

```diff
@@ -182,7 +182,7 @@
         {
           "column": "state_id",
           "subtype": "string_lastchars",
-          "constraint": "5"
+          "constraint": "3"
```

Comment:
Need to verify this, but it should be safe (needed here because our test states only have 3 letters). Even though the partition is changing it shouldn't recreate the triggers automatically

Reply:
confirmed that this won't be recreated automatically